yope-careers-service/src/main/java/io/yope/careers/visitor/CandidateVisitor.java

```diff
@@ -0,0 +1,60 @@
+package io.yope.careers.visitor;
+
+import io.yope.careers.domain.User;
+import io.yope.ethereum.visitor.BlockchainVisitor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@AllArgsConstructor
+@Getter
+public class CandidateVisitor extends BlockchainVisitor {
+
+    User user;
+
+    @Override
+    public String getContractKey() {
+        return "CandidateContract";
+    }
+
+    @Override
+    public String getContractFile() {
+        return "CandidateContract.sol";
+    }
+
+    @Override
+    public String getContractContent() {
+        return removeLineBreaks(getContractFile());
```

Comment:
It is in: https://github.com/yopeio/ethereum-api/blob/cc5373a03f4a4c1eb97e26e471d4fa3ba1bad6ba/yope-ethereum-model/src/main/java/io/yope/ethereum/utils/EthereumUtil.java#L11